broker/pkg/storage/cockroach/cockroach.go

```diff
@@ -346,13 +346,13 @@ func (c *CRClient) Delete(ctx context.Context, timelineID []byte, messageIDs []t
 			failBatch(err)
 			continue
 		}
-		_, err = txn.ExecContext(ctx, strings.Replace(query, "$TABLE", table(string(timelineID)), -1), args...)
+		_, err = txn.ExecContext(ctx, strings.Replace(query, "$TABLE", table(string(deleteMessages.TimelineID)), -1), args...)
```

Comment:
this will create a new allocation, we need to make a method that uses unsafe for conversion